<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestEnvPage</class>
 <widget class="QWidget" name="TestEnvPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>Test Environment Setup</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18pt; font-weight: bold;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="checklistPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="instructionLabel">
         <property name="text">
          <string>Please follow the steps below to set up the necessary testing frameworks for your project's technology stack.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="taskGroupBox">
         <property name="title">
          <string>Step X of Y: Task Name</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,0">
          <item>
           <widget class="QTextEdit" name="taskInstructionsTextEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="helpTextEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="doneButton">
              <property name="text">
               <string>Done, Next Step</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="helpButton">
              <property name="text">
               <string>I Need Help</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ignoreButton">
              <property name="text">
               <string>Ignore &amp; Continue</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="finalConfirmPage">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="successLabel">
         <property name="text">
          <string>All setup steps have been actioned.</string>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 14pt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="finalInstructionLabel">
         <property name="text">
          <string>Please confirm the final command that should be used to run all automated tests for this project (e.g., 'pytest', 'mvn test').</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="testCommandLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="finalizeButton">
         <property name="text">
          <string>Finalize Test Environment Setup</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>